src/content/docs/accounts/original-accounts-billing/original-users-roles/user-migration.mdx

```diff
@@ -13,26 +13,51 @@ Starting April 12, 2021, we're allowing some customers who have users on our ori
 
 ## Background [#background]
 
-On July 30 2020, we released a new and improved account/user model called the New Relic One user model. The newer model offers a much simpler, more efficient way to manage users and what they have access to ([learn more about the benefits of the new model](https://blog.newrelic.com/product-news/user-management-and-access-security/)). 
+On July 30 2020, we released a new and improved account/user model called the New Relic One user model. This newer model offers a simpler, more efficient way to manage users and what they have access to.
 
-At first, this new model was available mainly to new customers, and our existing customers' users remained on our original user model. Now some of our original-model customers can use a migration wizard in the UI to convert their users to the new model. When the migration process is complete, your users will be on the New Relic One user model and you’ll have [new procedures for managing your users](/docs/accounts/accounts-billing/new-relic-one-user-management/introduction-managing-users/). 
+At first, this new model was available mainly to new customers, and our existing customers' users remained on our original user model. Now some of our original-model customers can use a migration wizard in the UI to convert their users to the new model. When the migration process is complete, your users will be on the New Relic One user model and you’ll have [new procedures for managing your users](/docs/accounts/accounts-billing/new-relic-one-user-management/introduction-managing-users/).  
```

Comment:
Is that first sentence still true? Is the new model still available "mainly to new customers"?